Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3.8.19-slim-bullseye
```

this is quite an old and specific version of python, can something newer be used? given 3.8 will be EOL in October.
```suggestion
FROM python:3.10-slim-bullseye
```

---

This for example can be a variable, each release many images will be created (one per python version).